src/components/Posts/PostMenu.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  IconButton,
+  ListItemIcon,
+  ListItemText,
+  Menu,
+  MenuItem,
+} from "@mui/material";
+import React, { useRef, useState } from "react";
+import { useAppDispatch, useAppSelector } from "../../state/hooks";
+import { MoreVert, Edit, Delete, Link } from "@mui/icons-material";
+import axios from "axios";
+import { deletePost } from "../../state/slices/postsSlice";
+import { useNavigate } from "react-router-dom";
+import PostDeleteModal from "./PostDeleteModal";
+
+type PostMenuProps = {
+  authorId: number;
+  postId: number;
+  isExpandedPost?: boolean;
+};
+
+const styles = {
+  icon: {
+    color: "black.main",
+  },
+  listItemText: { fontWeight: "bold" },
```

Comment:
We can get rid of this style now that we're using the typography variants

Reply:
removed